android/app/src/main/java/com/salesforce/reactnativetemplate/MainActivity.java

```diff
@@ -24,55 +24,29 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-package com.salesforce.samples.myuserpicreactnative;
-
-import android.Manifest;
-import android.annotation.TargetApi;
-import android.content.pm.PackageManager;
-import android.os.Build;
+package com.salesforce.reactnativetemplate;
 
 import com.salesforce.androidsdk.reactnative.ui.SalesforceReactActivity;
 
 public class MainActivity extends SalesforceReactActivity {
 
-    private static final int REQUEST_CODE_ASK_PERMISSIONS = 123;
-
-    @Override
-    public void onResume() {
-        super.onResume();
-
-        // On N, permissions are granted at run time
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
-            ensurePermission(Manifest.permission.CAMERA);
-            ensurePermission(Manifest.permission.WRITE_EXTERNAL_STORAGE);
-        }
-    }
-
-    @TargetApi(Build.VERSION_CODES.M)
-    private void ensurePermission(String permission) {
-        if (this.checkSelfPermission(permission) != PackageManager.PERMISSION_GRANTED) {
-            requestPermissions(new String[] {permission}, REQUEST_CODE_ASK_PERMISSIONS);
-        }
-    }
-
     /**
      *
      * @return true if you want login to happen when application launches
      *         false otherwise
      */
-    @Override
-    public boolean shouldAuthenticate() {
-        return false;
-    }
-
+	@Override
+	public boolean shouldAuthenticate() {
+		return true;
+	}
 
-    /**
+	/**
 	 * Returns the name of the main component registered from JavaScript.
 	 * This is used to schedule rendering of the component.
 	 */
 	@Override
 	protected String getMainComponentName() {
-		return "MyUserPicReactNative";
+		return "ReactNativeTemplate";
```

Comment:
Should be `MyUserPicReactNative`.

Reply:
I ran forcereact to regenerate the app - I will check if there is a bug there.

Reply:
I ran forcereact to regenerate the app - I will check if there is a bug there.

Reply:
Done.